libs/java/auth_core/src/main/java/com/yahoo/athenz/auth/util/GlobStringsMatcher.java

```diff
@@ -30,7 +29,7 @@
     private final List<String> patterns;
 
     public GlobStringsMatcher(String systemProperty) {
-        List<String> globList = ZTSUtils.splitCommaSeperatedSystemProperty(systemProperty);
+        List<String> globList = AthenzUtils.splitCommaSeperatedSystemProperty(systemProperty);
```

Comment:
we should update the debug statement on line 35 below. it's referring to Ignored services - we should just include the property name in the statement